actionpack/lib/action_controller/metal/data_streaming/disposition_filename_parameters.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ActionController
+  module DataStreaming
+    class DispositionFilenameParameters # :nodoc:
```

Comment:
If we want to reuse inside Active Storage I don't think it should be inside `DataStreaming`. `ActionController:: DispositionFilename` is a better name.

Reply:
okay, I'll rename

Reply:
ah, you also meant `DispositionFilenameParameters` should be renamed to `DispositionFilename`, right?

Reply:
Right, but maybe `DispositionFilenameParameter` is better.